docs/tutorials/command_line_client.md

```diff
@@ -42,7 +42,7 @@ synapse [-h] [--version] [-u SYNAPSEUSER] [-p SYNAPSEPASSWORD] [-c CONFIGPATH] [
 |---------------------|---------|---------------------------------------------------------------------------|-----------------------|
 | `--version`         | Flag    | Show program’s version number and exit                                    |                       |
 | `-u, --username`    | Option  | Username used to connect to Synapse                                       |                       |
-| `-p, --password`    | Option  | Password, api key, or token used to connect to Synapse                    |                       |
+| `-p, --password`    | Option  | Auth Token used to connect to Synapse                                     |                       |
```

Comment:
Nit: On the synapse docs, we refer to the Auth Token as Personal Access Token.  We unfortunately have SYNAPSE_AUTH_TOKEN everywhere, lets make sure we are intentional about calling out that linkage.

Reply:
I added some consistent verbiage around this to reference both terms:

https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1041/commits/c70f8223857f23332c03765c20ddf22ed046b7ef